pkg/webui/components/dropdown/dropdown.styl

```diff
@@ -20,18 +20,18 @@ ul.dropdown
   background: $c.white
   box-shadow: 0px 3px 16px 0px rgba(0, 0, 0, 0.06)
   position: absolute
-  padding: $cs.s
-  min-width: 14rem
+  padding: $cs.xs
+  width: max-content
+  min-width: 16rem
+  max-width: 24rem
   z-index: $zi.dropdown
   margin: $cs.xxs 0 0
   opacity: 0
   visibility: hidden
-  transition: opacity 0.2s ease-in-out, visibility 0s linear 0.2s
```

Comment:
I've removed all transitions for now, as I believe the component behaves better without.